.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default reviewers for all pull requests to this repo.
+* @kopepasah
```

Comment:
We should probably discuss this, as we removed the CODEOWNERS file in the past.

Reply:
Okay, that makes sense. My opinion would be that for OSS repos, someone needs to own the code. Regardless if we have an official CODEOWNERS file or not, having someone as a stakeholder ensures the software does not go stale.

Reply:
Agreed. When we had a CODEOWNERS file, we had almost all the devs in it, I think ;)

Reply:
Another reason was the review reminder notifications in Slack. We got lazy assigning the proper people so everyone was pinged for everything and the result was slacking on reviews and stories dropping.

I think this is fine to keep if you want since it's a small project and this kind of stuff is where you are wanting to be focused.

Reply:
Thanks for the feedback! I will remove the file, actually. Unless we come up with a solution as a team to use this file, let's keep it out of our repos.